ground/src/main/java/com/google/android/ground/repository/SurveyRepository.kt

```diff
@@ -65,9 +65,11 @@ constructor(
       else getOfflineSurvey(it).map { s -> Optional.of(s) }
     }
 
-  var lastActiveSurveyId: String = ""
+  var lastActiveSurveyId: String
```

Comment:
While you're at it can you please rename this to `activeSurveyId`, since it refers to the current state, not previous one?